test/Dialect/SMT/bitvectors.mlir

```diff
@@ -0,0 +1,13 @@
+// RUN: circt-opt %s | circt-opt | FileCheck %s
+
+// CHECK-LABEL: func @bitvectors
+func.func @bitvectors() {
+  // CHECK: %c5_bv32 = smt.bv.constant #smt.bv<5> : !smt.bv<32> {smt.some_attr}
+  %c5_bv32 = smt.bv.constant #smt.bv<5> : !smt.bv<32> {smt.some_attr}
+  // CHECK: %c92_bv8 = smt.bv.constant #smt.bv<92> : !smt.bv<8> {smt.some_attr}
+  %c92_bv8 = smt.bv.constant #smt.bv<0x5c> : !smt.bv<8> {smt.some_attr}
+  // CHECK: %c-1_bv8 = smt.bv.constant #smt.bv<-1> : !smt.bv<8>
```

Comment:
I'm also wondering, should we print this as `-1`, or should we print this as `255`?
I do not have a good answer, but that's probably a question we should ask ourselves.

Reply:
I just went with the same as the HW dialect does. In the end it's a signless bitvector, whether we interpret it here as a signed or unsigned value is probably equally good or bad.